core/sql/bin/SqlciErrors.txt

```diff
@@ -212,6 +212,9 @@
 1210 ZZZZZ 99999 UUUUUUUU UUUUU UUUUUUU Column $0~String0 is not allowed at position $1~Int0 in the SPLIT BY clause, because SPLIT BY must specify a prefix of the clustering key columns and the next clustering key column is $2~String1.
 1211 ZZZZZ 99999 UUUUUUUU UUUUU UUUUUUU The column values of the SPLIT BY first key in position $0~Int0 (one-based) are not greater (for ascending columns) or less (for descending columns) than its predecessor.
 1212 ZZZZZ 99999 UUUUUUUU UUUUU UUUUUUU An HBase region start key provided only part of the value for Trafodion key column $0~Int0. This could in some unusual situations cause errors.
+1213 ZZZZZ 99999 UUUUUUUU UUUUU UUUUUUU Unable to create or initialize a connection to Apache Hive.
```

Comment:
Yes, very good point. This is a security hole. A safer way is to do it is to use volatile tables, which are not yet implemented. I filed [TRAFODION-2316](https://issues.apache.org/jira/browse/TRAFODION-2316) for this issue. I also should have made that very clear in the description of this checkin - will fix that for the "real" pull request.